core-blocks/code/index.js

```diff
@@ -14,7 +14,7 @@ export const name = 'core/code';
 export const settings = {
 	title: __( 'Code' ),
 
-	description: __( 'The code block maintains spaces and tabs, great for showing code snippets.' ),
+	description: __( 'Add text that respects your spacing and tabs -- perfect for displaying code.' ),
```

Comment:
>Add text that respects your spacing and tabs -- perfect for displaying code.

Instead of a faux-em dash, should we use a real em dash `—` ?

Or am I mistaken in my understanding of `--` being more of a convenience since keyboards don't have intended character readily available?

A few other instances of this in the same pull request.

Reply:
I fixed one of them in https://github.com/WordPress/gutenberg/pull/7196, it would be nice to catch them all.